pkgs/development/python-modules/polars/default.nix

```diff
@@ -66,7 +64,6 @@ buildPythonPackage {
   # ];
 
   meta = with lib; {
-    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
I believe you wanted `stdenv.isLinux && !stdenv.isAarch64`

Reply:
It did build on all platforms.  It's probably something from recent staging-next merge. I'll have a look at it today

Reply:
Thanks, appreciate that! looking into possibly bumping this further, current version is 0.18.13.

Reply:
Yes I was waiting for rust 1.71